python/pyspark/sql/connect/functions.py

```diff
@@ -2467,7 +2467,7 @@ def udf(
 
 
 def pandas_udf(*args: Any, **kwargs: Any) -> None:
```

Comment:
what about adding a few comments here? So that the contributors will be aware that this is already implemented

Reply:
+1 for @zhengruifeng 's comment.

Reply:
Sounds good! Added a comment.